### PR TITLE
Configure github actions

### PR DESCRIPTION
**Pull request by VBeytok**
To have commit message like `"Updated %language%"` for the language, need to create [step](https://github.com/purrbot-site/PurrBot/compare/feature/add-crowdin-action...VBeytok:feature/add-crowdin-action?expand=1#diff-719333b616448c4150fe3848e76b55fbb9864ef46fe9543509fef22ca2cef2acR23) and add [option](https://github.com/purrbot-site/PurrBot/compare/feature/add-crowdin-action...VBeytok:feature/add-crowdin-action?expand=1#diff-719333b616448c4150fe3848e76b55fbb9864ef46fe9543509fef22ca2cef2acR49) `download_language: 'fr'` with commit message `"Updated French"`.

### CODE REVIEW

File: .github/workflows/crowdin_export.yml [hunk 1]
```diff
@@ -16,15 +16,17 @@ jobs:
     steps:
       - name: "Update Code"
         uses: actions/Checkout@v2
+        
       - name: "Export Crowdin Translations"
-        uses: crowdin/github-action@v1.0.20
+        uses: crowdin/github-action@1.0.21
         with:
-          upload_sources: true # Upload sources
+          crowdin_branch_name: 'master'
```
Comment (Andre601) [inline .github/workflows/crowdin_export.yml:23]: Is that even needed? I would assume it defaults to that branch so why apply this?

**Comment by Andre601** [comment]
Not really what I want. This seems to be more about downloading (Importing) a specific language, while I just want to have the same as #303 
Thanks for the contribution, but this seems to be exactly what I DON'T need to have.

**Comment by Andre601** [comment]
Additionally is this exactly the reason for my Issue on the GitHub Repo: The action lacks crucial parts that would allow a basic migration from the setup with a GitHub Account to one with GitHub Actions.

**Comment by VBeytok** [comment]
It's so sad to hear that. Crowdin Github Action work on Crowdin CLI and it can build all languages or just one, currently..

**Pull request closed without merging**